conda-store-server/conda_store_server/plugins/hookspec.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) conda-store development team. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+from collections.abc import Iterable
+
+import pluggy
+
+from conda_store_server.plugins.types import LockPlugin
+
+spec_name = "conda-store-server"
```

Comment:
Should we just have `conda-store` here? I think we were supposed to merge them at some point, so I don't know if it makes sense to introduce this distinction here. WDYT @trallard?

Reply:
I'm +1 to renaming this to `conda-store`